frontend/src/store/actions/authActions.js

```diff
@@ -1,10 +1,10 @@
-import axios from '../plugins/axios';
+import axios from '../../plugins/axios';
```

Comment:
can we use aliases? 
@plugins/axios
Check with Ewelina.

Reply:
please check my comment in last PR about axios, you should be able to just import it from library not from plugins

Reply:
@ alias seems to be problematic to achieve but i managed to create imports with base url of src folder so it is looks like this 'plugins/axios' for now

Reply:
also changed axios plugin

Reply:
@dawid-pstrak-rmtcfm-com but have you done it as Ewelina said?

Reply:
and as for aliases you can check this https://stackoverflow.com/a/63068531 , you can use craco package for that https://www.npmjs.com/package/@craco/craco

Reply:
Ok saw this but i though it is not worth to use external package for this. I think i ll do this on next PR because here is quite a ot of changes. For now I am going to push current work, also please take a look on this axios plugin because i don;t know if i understood your think correclty